code/modules/surgery/bodyparts/robot_bodyparts.dm

```diff
@@ -100,6 +100,15 @@
 
 	damage_examines = list(BRUTE = ROBOTIC_BRUTE_EXAMINE_TEXT, BURN = ROBOTIC_BURN_EXAMINE_TEXT, CLONE = DEFAULT_CLONE_EXAMINE_TEXT)
 
+/obj/item/bodypart/leg/left/robot/emp_act(severity)
+	. = ..()
+	if(!.) // other emp effects handled in parent
```

Comment:
Comment (here and elsewhere) probably not necessary, anyone reading this should know that's just what `..()` does.

Reply:
removed